charts/k8ssandra/templates/prometheus/service_monitor.yaml

```diff
@@ -14,6 +19,9 @@ spec:
     matchExpressions:
       - {key: cassandra.datastax.com/cluster, operator: In, values: [{{ include "k8ssandra.clusterName" . }}]}
       - {key: cassandra.datastax.com/datacenter, operator: In, values: [{{ include "k8ssandra.datacenterName" . }}]}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace | quote }}
```

Comment:
This can be simplified as `{{ .Release.Namespace }}`. No need for the `quote` function.

Reply:
gotcha 👍